### DISCUSSION TITLE
Welcome to SLY discussions!

### DISCUSSION THREAD

**Discussion by joaotavora** [joaotavora/sly#390] [General]
## 👋 Welcome!

Since SLY doesn't have a mailing list, this is the place for general comments and questions about SLY that aren't worth making an issue.  Maybe a good place for the perennial "SLY or SLIME" question? Be polite, objective, etc etc.

EDIT: by editing this template, I seem to have made a first meta-discussion.

**Comment by svetlyak40wt** [comment]
Great!
